src/Connector.Snowflake/SnowflakeConnectionData.cs

```diff
@@ -0,0 +1,54 @@
+﻿using CluedIn.Connector.Common.Configurations;
+using CluedIn.Core.Connectors;
+using System;
+using System.Collections.Generic;
+
+namespace CluedIn.Connector.Snowflake
+{
+    public class SnowflakeConnectionData : IConnectorConnection
```

Comment:
this way, without inheritance from `CrawlerJobData` class is more lightweight. So it's easier to save it into cache storage.